### PR TITLE
Don't panic on invalid plugin id

### PR DESCRIPTION
**Pull request by zshipko**

### CODE REVIEW

**Review by bhelx — APPROVED** [review]
Much appreciated! If possible we should probably ban panics in the runtime.

File: runtime/src/export.rs [hunk 1]
```diff
@@ -1,6 +1,22 @@
 /// All the functions in the file are exposed from inside WASM plugins
 use crate::*;
 
+// This macro unwraps input arguments to prevent functions from panicking,
+// it should be used instead of `Val::unwrap_*` functions
+macro_rules! get {
```
Comment (nilslice) [inline runtime/src/export.rs:6]: nit: maybe a more descriptive name here: `get_val`? `get_offset`?

File: runtime/src/context.rs [hunk 2]
```diff
@@ -57,6 +57,7 @@ impl Context {
 
     /// Set the context error
     pub fn set_error(&mut self, e: impl std::fmt::Debug) {
+        trace!("Set context error: {:?}", e);
```
Comment (nilslice) [inline runtime/src/context.rs:60]: would it make sense to use `error!` here? or do we likely want to catch that at the source of the error instead (and  using `trace!` here is best for the intentionally verbose debugging level?)

**Review by nilslice — APPROVED** [review]
Great as-is, just a couple questions/nits. If you think it's good to go, merge away!

File: runtime/src/context.rs (hunk 2 above)
Comment (zshipko) [reply runtime/src/context.rs:60]: I believe the error is being logged somewhere else, this log was added while debugging some issues with the error message itself.

**Pull request merged**